lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -349,6 +350,10 @@ defmodule ExUnit.CLIFormatter do
         config.skipped_counter > 0,
         &(&1 <> ", " <> skipped("#{config.skipped_counter} skipped", config))
       )
+      |> if_true(
+        config.excluded_counter > 0,
+        &(&1 <> ", #{config.excluded_counter} excluded")
```

Comment:
WDYT about this?

```suggestion
        &(&1 <> " (#{config.excluded_counter} excluded)")
```

Then it will be very clear it is not part of the overall sum?

Reply:
I dig it! Made that change 🤙